config/packages/cache.yaml

```diff
@@ -1,5 +1,6 @@
 framework:
     cache:
+        directory: '%sulu.common_cache_dir%'
```

Comment:
This should prepend by us because other caches like apc and redis which is common use in production behave the same.

https://github.com/sulu/sulu/pull/5180/files